README.md

```diff
@@ -17,7 +17,15 @@
 ```
 
 Start the rails app
+   There are two methods to starting the rails app.  One is to run this command directly which will take care of some of the extra steps you would have to do otherwise:
   ```
+  $ bin/dev
+  ```
+  OR run the following commands:
+  
+  ```
+  $ yarn install
+  $ yarn build:css
```

Comment:
I think it'd be good to list the things `bin/dev` actually does, which are:
- start a rails server (the same as `rails server`)
- run `yarn build:css` with the `--watch` option, so it re-runs when you change CSS files

(I learned that by looking at `Procfile.dev`, which is what gets invoked by `bin/dev`.)

And then you can say "or you can invoke these tasks separately with..." etc. and list them out

Reply:
Maybe just point to Procfile.dev and then we don't have the README getting out of sync with what is actually invoked.

Reply:
I'd also urge us to drop the "two ways" (there are actually more than two) and just do `bin/dev` Then there's no confusion.

Reply:
And `yarn install` is always required to install the dependencies (just like `bundle install`) however, it's not required every time you start the server.